front-end/src/MyShows.js

```diff
@@ -14,6 +14,13 @@ import Modal from 'react-modal'
 import Select from 'react-select'
 import { Link, useHistory } from 'react-router-dom'
 import { platforms } from './DropdownOptions'
+import amazon from './Images/amazon.jpg'
+import crunchyroll from './Images/crunchyroll.png'
+import disney from './Images/disney.jpg'
+import hbo from './Images/hbo.jpg'
+import hulu from './Images/hulu.png'
+import netflix from './Images/netflix.png'
+import other from './Images/other.png'
```

Comment:
I don't think other.png exists currently; if you added it, you might not have staged it for commit.